packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 - We updated version of `@wdio` and `react-native` libraries.
 - We changed `eslint-plugin-react` to version `7.28.0` in order to fix this [issue](https://github.com/yannickcr/eslint-plugin-react/issues/3215).
+- We increased the max SVG file size limit to 200 kB for the `editor config`.
```

Comment:
Maybe elaborate on what the change can be used for, e.g. to accommodate SVGs of a bigger size used in structure preview mode.

Reply:
I updated. What do you think about this one?

Reply:
It's good, nice one.